index.html

```diff
@@ -0,0 +1,102 @@
+            Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects;
+           </p>
+        </div>
+       <div class="main-section">
+           <div id="javascript_and_java" >Javascript and Java</div>
+                <p>
+                    JavaScript and Java are similar in some ways but fundamentally different in some others. 
+                    The JavaScript language resembles Java but does not have Java's static typing and strong type checking. 
+                    JavaScript follows most Java expression syntax, naming conventions and basic control-flow 
+                    constructs which was the reason why it was renamed from LiveScript to JavaScript.
+                </p>
+                <p>
+                    In contrast to Java's compile-time system of classes built by declarations, 
+                    JavaScript supports a runtime system based on a small number of data types representing numeric, 
+                    Boolean, and string values. JavaScript has a prototype-based object model instead of the more common 
+                    class-based object model. The prototype-based model provides dynamic inheritance; that is, 
+                    what is inherited can vary for individual objects. JavaScript also supports functions 
+                    without any special declarative requirements. 
+                    Functions can be properties of objects, executing as loosely typed methods
+                </p>
+                <ul>
+                    <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.
+                    </li>
+                    <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application, or perform file manipulations on a server.
+                    </li>
+                </ul>
+            </div>
+       <div class="main-section">
+          <div id="HTML"> Hypertext MarkUp Language</div>
+          <p>
+            HyperText Markup Language (HTML) is the basic scripting language used by web browsers to render pages on the world wide web.
+        </p>
+        <p>
+            HyperText allows a user to click a link and be redirected to a new page referenced by that link
+
+        </p>
+        <ul>
+            <li>webpage skeletal appearanc </li>
+        </ul>
+    </div>
+       <div class="main-section">
+           <div id="CSS">Cascading Style Sheet</div>
```

Comment:
I think it would look nice if you could create a new div that encapsulates the paragraph

Reply:
- [ ] Hey @brainconnect93, Great job with this recommendation but you need to provide a more genuine reason on why you request your changes so the reviewee is convinced and improves. Here you can tell the reviewee how adding a div container can help adding padding and margins to grouped elements easier. Or you can even go on to suggest a semantic tag to improve the semantic flow of the document :)